main/test/ca/uwaterloo/flix/tools/TestBootstrap.scala

```diff
@@ -11,34 +12,46 @@ import java.util.zip.ZipFile
 import scala.jdk.CollectionConverters.EnumerationHasAsScala
 import scala.util.Using
 
-class TestPackager extends FunSuite {
+class TestBootstrap extends FunSuite {
 
   private val ProjectPrefix: String = "flix-project-"
 
   private val DefaultOptions: Options = Options.Default
 
   test("init") {
     val p = Files.createTempDirectory(ProjectPrefix)
-    Packager.init(p, DefaultOptions)
+    Bootstrap.init(p, DefaultOptions)
   }
 
   test("check") {
     val p = Files.createTempDirectory(ProjectPrefix)
-    Packager.init(p, DefaultOptions)
-    Packager.check(p, DefaultOptions)
+    Bootstrap.init(p, DefaultOptions)
+    val b = Bootstrap.bootstrap(p)(System.out) match {
+      case Result.Ok(bootstrap) => bootstrap
+      case Result.Err(e) => fail(e.toString)
+    }
+    b.check(DefaultOptions)
   }
 
   test("build") {
     val p = Files.createTempDirectory(ProjectPrefix)
-    Packager.init(p, DefaultOptions)
-    Packager.build(p, DefaultOptions)
+    Bootstrap.init(p, DefaultOptions)
+    val b = Bootstrap.bootstrap(p)(System.out) match {
+      case Result.Ok(bootstrap) => bootstrap
+      case Result.Err(e) => fail(e.toString)
+    }
+    b.build(DefaultOptions)
   }
 
   test("build-jar") {
     val p = Files.createTempDirectory(ProjectPrefix)
-    Packager.init(p, DefaultOptions)
-    Packager.build(p, DefaultOptions)
-    Packager.buildJar(p, DefaultOptions)
+    Bootstrap.init(p, DefaultOptions)
+    val b = Bootstrap.bootstrap(p)(System.out) match {
+      case Result.Ok(bootstrap) => bootstrap
+      case Result.Err(e) => fail(e.toString)
+    }
+    b.build(DefaultOptions)
+    b.buildJar(DefaultOptions)
```

Comment:
The idiomatic style for these tests would simply be:

```
    Bootstrap.init(p, DefaultOptions)
    val b = Bootstrap.bootstrap(p)(System.out).get
    b.build(DefaultOptions)
    b.buildJar(DefaultOptions)
```

Obviously calling `get` is unsafe -- and should never be done in production code-- but its perfectly fine in test cases (for clarity).